editor/src/java/com/dynamo/upnp/SSDP.java

```diff
@@ -46,7 +38,7 @@ public boolean connect(NetworkInterface networkInterface) throws IOException {
             if (!addresses.isEmpty()) {
                 mcastSocket = new MulticastSocket(SSDP_MCAST_PORT);
                 mcastSocket.setNetworkInterface(networkInterface);
-                mcastSocket.joinGroup(SSDP_MCAST_ADDR);
+                mcastSocket.joinGroup(new InetSocketAddress(SSDP_MCAST_ADDR, 0), null);
```

Comment:
Fixed another deprecation warning. JDK's javadocs contained this snippet with the equivalent undeprecated code.